velox/exec/ContainerRowSerde.h

```diff
@@ -32,18 +32,35 @@ class ContainerRowSerde {
   static void
   deserialize(ByteStream& in, vector_size_t index, BaseVector* result);
 
+  /// Returns < 0 if 'left' is less than 'right' at 'index', 0 if
+  /// equal and > 0 otherwise. flags.nullHandlingMode can be only NoStop and
+  /// support null-safe equal.
   static int32_t compare(
       ByteStream& left,
       const DecodedVector& right,
       vector_size_t index,
       CompareFlags flags);
 
+  /// Returns < 0 if 'left' is less than 'right' at 'index', 0 if
+  /// equal and > 0 otherwise. flags.nullHandlingMode can be only NoStop and
+  /// support null-safe equal.
   static int32_t compare(
       ByteStream& left,
       ByteStream& right,
       const Type* type,
       CompareFlags flags);
 
+  /// Returns < 0 if 'left' is less than 'right' at 'index', 0 if
+  /// equal and > 0 otherwise. If flags.nullHandlingMode is StopAtNull,
+  /// returns std::nullopt if either 'left' or 'right' value is null or contains
+  /// a null. If flags.nullHandlingMode is NoStop, returns
```

Comment:
> If flags.nullHandlingMode is NoStop, returns
> std::optional<int32_t>(0) if both sides are nulls

Let's remove this sentence. It is missing in comments for other versions of this method.

Alternatively, let's say that "If flags.nullHandlingMode is NoStop then NULL is considered equal to NULL."